grammars/php.cson

```diff
@@ -1385,14 +1385,6 @@
     # http://www.php.net/manual/en/language.types.string.php#language.types.string.parsing
     'patterns': [
       {
-        'match': '\\\\[0-7]{1,3}'
```

Comment:
Please revert these and rename their scopes to `constant.character.escape.octal.php` and `constant.character.escape.hex.php` respectively. While you are at it, please also consider adding the unicode escape highlighting that can be found from the link above.

Reply:
Ah, so that's what these were.  I assumed that the other pattern would take care of them.  Will add them back and add some specs.

Reply:
The appropriate table is located at http://ee1.php.net/manual/en/language.types.string.php#language.types.string.syntax.double